### ISSUE TITLE
Add tutorial doc

### ISSUE THREAD

**Issue by danielhoherd** [quintush/helm-unittest#118]
I have written [a tutorial doc on how to write unittests for this helm-unittest framework](https://github.com/astronomer/airflow-chart/blob/31634a60610/tests/README.md). It has been useful to both my company and to FOSS contributors. I'd like to contribute it here, but I'm not sure where it would fit in.

**Comment by quintush** [comment]
Hello @danielhoherd,

If you want, you can add the tutorial as a file to the root folder (calling it TUTORIAL.md).
In the README.md I would prefer to append the link of the tutorial in the "Get Started" section, we you can decribe a more elaborated explanation to get started.

**Comment by danielhoherd** [comment]
@quintush We're actually not using the helm-unittest framework anymore, so I'm not sure how relevant that tutorial is to modern helm-unittest. We [switched to using pytest to test helm charts](https://github.com/astronomer/astronomer/blob/master/tests/README.md) because it is more feature rich and is a more broadly useful skill to have, and is more broadly used. FWIW, your fork of helm-unittest was the best one I had used, and I appreciated the work you put into it.
